.cloud-build/execute_changed_notebooks_helper.py

```diff
@@ -110,6 +114,31 @@ def _process_notebook(
         nbformat.write(nb, new_file)
 
 
+def _get_notebook_python_version(notebook_path: str) -> str:
+    """
+    Get the python version for running the notebook if it is specified in
+    the notebook.
+    """
+    python_version = PYTHON_VERSION
+
+    file = open(notebook_path) # Open the ipynb file that is passed in
+    src = file.read() # Read the file in
+    j = json.loads(src) # Parse the JSON into an object
+
+    for cell in j['cells']: #Iterate over all the cells in the ipynb
+        if cell['cell_type'] == 'markdown':
+            markdown = str.join('', cell['source'])
+
+            # Look for the python version specification pattern
+            re_match = re.search('python\ ?version[:,=]\ ?\d(.\d)?', markdown, flags=re.IGNORECASE)
```

Comment:
Strange to use regex twice like this. Can't you just extract using the first regex?

Reply:
Done: cf86cf923aa4ed2fde3b91154911c3a3c0035dd1
Thank you for your help! :)